### PR TITLE
Fix camelCase functions in telegram.ext.ExtBot

### PR DESCRIPTION
**Pull request by OctoNezd**
### Checklist for PRs

- [x] Created new or adapted existing unit tests

### Bug description

camelCase aliases in telegram.ext.ExtBot still pointed to telegram.Bot. This PR fixes that and adds test_extbot_camelcase test that checks for this issue.

### CODE REVIEW

File: tests/test_extbot_camelcase.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+def to_camel_case(snake_str):
+    """https://stackoverflow.com/a/19053800"""
+    components = snake_str.split('_')
+    # We capitalize the first letter of each component except the first one
+    # with the 'title' method and join them together.
+    return components[0] + ''.join(x.title() for x in components[1:])
+
+
+class TestCamelCase:
+    def test_camel_case(self):
```
Comment (Bibo-Joshi) [inline tests/test_extbot_camelcase.py:13]: please move this into `test_bot.py`. We work with `ExtBot` there anyway. Actually, you can extend the test to run on both `tg.Bot` and `tg.ext.ExtBot` :)

Comment (OctoNezd) [reply tests/test_extbot_camelcase.py:13]: Why do this with tg.Bot though?

Comment (Bibo-Joshi) [reply tests/test_extbot_camelcase.py:13]: to make sure that all snake_case functions have a `camelCase` equivalent

Comment (OctoNezd) [reply tests/test_extbot_camelcase.py]: Implemented in https://github.com/python-telegram-bot/python-telegram-bot/pull/2659/commits/922c1e530e441373c34d1ba15d7521b87b60ac81

File: tests/test_bot.py [hunk 2]
```diff
@@ -2455,3 +2463,25 @@ def test_defaults_warning(self, bot, recwarn, cls, warn):
             assert 'Passing Defaults to telegram.Bot is deprecated.' in str(recwarn[-1].message)
         else:
             assert len(recwarn) == 0
+
+    def test_camel_case_redefinition_extbot(self):
+        invalid_camel_case_functions = []
+        for function_name, function in ExtBot.__dict__.items():
+            camel_case_function = getattr(ExtBot, to_camel_case(function_name), False)
+            if callable(function) and camel_case_function and camel_case_function != function:
```
Comment (Bibo-Joshi) [inline tests/test_bot.py:2471]:
```suggestion
            if callable(function) and camel_case_function and camel_case_function ist not function:
```
just a minor nit pick b/c I have bad experience with relying on `__eq__` to do what you expect it to do :D

File: tests/test_bot.py [hunk 3]
```diff
@@ -2455,3 +2463,25 @@ def test_defaults_warning(self, bot, recwarn, cls, warn):
             assert 'Passing Defaults to telegram.Bot is deprecated.' in str(recwarn[-1].message)
         else:
             assert len(recwarn) == 0
+
+    def test_camel_case_redefinition_extbot(self):
+        invalid_camel_case_functions = []
+        for function_name, function in ExtBot.__dict__.items():
+            camel_case_function = getattr(ExtBot, to_camel_case(function_name), False)
+            if callable(function) and camel_case_function and camel_case_function ist not function:
```
Comment (Bibo-Joshi) [inline tests/test_bot.py:2471]:
```suggestion
            if callable(function) and camel_case_function and camel_case_function is not function:
```
stupid german me …

**Comment by Bibo-Joshi** [comment]
Thanks for the contribution :)

**Pull request merged**